website/content/en/docs/User Guide/Cost Explained/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Breaking Down Cloud Operations Sandbox Cost"
+linkTitle: "Sandbox Cost Explained"
+weight: 2
+---
+{{% pageinfo %}}
+* [Cloud Operations Sandbox Cost Estimations](#Cloud-Operations-Sandbox-Cost-Estimations)
+  * [GCP Calculator](#GCP-Calculator)
+  * [Sandbox Cost Estimations](#Sandbox-Cost-Estimations)
+  * [Sandbox Compute Cost Estimations](#Sandbox-Compute-Cost-Estimations)
+  
+ 
+## Cloud Operations Sandbox Cost Estimations
+Cloud Operations Sandbox Cost is combined from compute cost, storage and usage of network and additional services like Cloud Operations Suite. While some of the cost, like GKE infrastructure is constant, for some services cost is a factor of usage. Note, that some of Sandbox costs fall under the [free tier](https://cloud.google.com/free/docs/gcp-free-tier#free-tier), but not not all.
+To learn more about billing and pricing please refer to: [Google Cloud Pricing Overview](https://cloud.google.com/pricing/), [Google’s Pricing philosophy](https://cloud.google.com/pricing/philosophy/), [GCP's Billing](https://cloud.google.com/billing/docs).
+
+To view billing information for your project in the console go to [billing.](https://console.cloud.google.com/billing)
+![image](/docs/images/user-guide/57-billing-menu.png)
+
+![image](/docs/images/user-guide/58-billing-console.png)
```

Comment:
I'd suggest adding a sentence of context before this image. Maybe "The GCP billing page will break down your expenses by resource usage"